src/graphql/queries/extensions.graphql

```diff
@@ -14,3 +14,13 @@ query GetVotingRepInstallations($votingRepHash: String!, $colonyAddress: ID!) {
     }
   }
 }
+
+query GetColonyExtensionsByColonyAddress($colonyAddress: ID!) {
+  listColonyExtensions(
```

Comment:
I think it's better to use `getExtensionByColonyAndHash` here (the hash is optional), since the `list` will list you the first 100 installations across all colonies, and then apply the colony-specific filter, meaning you could end with no results if there are enough colonies and installations in the db. 

Whereas `getExtensionByColonyAndHash` handles the filtering before the results are returned, meaning you're guaranteed to get what you're expecting.